src/shared/footer.mustache

```diff
@@ -111,5 +111,6 @@
 <!-- Start: Additional component(s) script -->
 <!-- ... -->
 <!-- End: Additional component(s) script -->
+<script src="/static/js/updateFrontend.js" defer></script>
```

Comment:
We laden dit nu op elke pagina, is dat de bedoeling?

Reply:
Ja, de functionaliteit is inmiddels generiek: Als de body een specifiek attribuut heeft gaat ie laden. Dat betekent dat we zo op de homepage ook zaken (of iets anders) kunnen laden.